public/app/plugins/datasource/elasticsearch/configuration/ElasticDetails.tsx

```diff
@@ -75,6 +75,17 @@ export const ElasticDetails = (props: Props) => {
           />
         </div>
 
+        <div className="gf-form max-width-25">
+          <FormField
+            labelWidth={10}
+            inputWidth={15}
+            label="Custom Query Parameters"
```

Comment:
To make this explicit for search query parameters and nothing else, maybe consider naming this Custom Search Query Parameters?